packages/react-avatar/README.md

```diff
@@ -2,16 +2,43 @@
 
 **React Avatar components for [Fluent UI](https://dev.microsoft.com/fluentui)**
 
+The Avatar component represents a person or entity. It displays the person's image, initials, or an icon, and can be either circular or square.
```

Comment:
nit: TMP also renders hexagonal avatar. There is no support for that in the library itself but v0 has a docsite prototype showing how to achieve that using a wrapping `svg` with a `clipPath`: 
It might be a useful exercise to verify this with the converged avatar

Reply:
There already is a hexagonal avatar in storybook examples 👍